RM_SUI/RM_SUI/ContentView.swift

```diff
@@ -8,9 +8,92 @@
 import SwiftUI
 
 struct ContentView: View {
+    private enum Constants {
+        static let defaultColor: Color = .accentColor
+        static let exampleAlertTextName = "Example Alert"
+        static let exampleActionSheetTextName = "Example ActionSheet"
+        static let showTextName = "Show"
+        static let titleTextName = "Title"
+        static let exampleOneTextName = "Example Alert\nwith button and logic"
+        static let changeColorTextName = "Change Color"
+        static let emptyString = ""
+        static let okTextName = "Ok"
+        static let messageTextName = "Message"
+        static let clearColorTextName = "Clear Color?"
+        static let blackColorTextName = "Black Color"
+    }
+    @State var isExampleOneAlertShown = false
+    @State var isExampleTwoAlertShown = false
+    @State var labelColor = Constants.defaultColor
+    @State var isExampleOneSheetShown = false
+    @State var isExampleTwoSheetShown = false
+    
     var body: some View {
-        VStack {
-            Text("")
+        VStack(spacing: 10) {
```

Comment:
```suggestion
        VStack(spacing: 10) {
```
я бы вынес в константы

Reply:
Вынести числовые константы